makefile

```diff
@@ -195,31 +195,64 @@ documentation/stats-dangling.md: curation/nlp-matches.sssom.tsv
-$(SNOMED_OWL_DIR)/snomed-reasoned.ofn: $(SNOMED_OWL_DIR)/snomed-unreasoned.ofn
-	$(call robot_reason,$<,$@)
+#$(SNOMED_OWL_DIR)/snomed-reasoned.ofn: $(SNOMED_OWL_DIR)/snomed-unreasoned.ofn
+#	$(call robot_reason,$<,$@)
+
+# Extract ancestors: See https://robot.obolibrary.org/extract (ChEBI goal in this makefile also has more info)
+$(LOINC_SNOMED_OWL_DIR)/related-snomed-ancestor-closure.owl: $(LOINC_SNOMED_OWL_DIR)/snomed-with-loinc-snomed-full-unreasoned.owl $(LOINC_SNOMED_OWL_DIR)/loinc-snomed-module-classes-and-props.txt
+	robot extract --method BOT \
+	--input $(LOINC_SNOMED_OWL_DIR)/snomed-with-loinc-snomed-full-unreasoned.owl \
+	--term-file $(LOINC_SNOMED_OWL_DIR)/loinc-snomed-module-classes-and-props.txt --output $@
 
 # -- LOINC-SNOMED representation
-# Todo: if needed temporarily to make subclass rels / labels until this is ready, use instead as source: output/analysis/snomed/bak/snomed-parts-reasoned.owl instead of loinc-snomed-reasoned.owl
-# Todo: Structure this properly https://github.com/loinc/comp-loinc/issues/194
-# Todo: Change SNOMED-LOINC representation: (1) build SNOMED, (2) subset it by LOINC mappings
-# Todo: - how subset those mappings? involves mapped parts and terms. check notes
-# Todo: robot merge? What other inputs does this need?
-$(LOINC_SNOMED_OWL_DIR)/loinc-snomed-unreasoned.owl: $(DEFAULT_BUILD_DIR)/snomed-parts.owl | $(LOINC_SNOMED_OWL_DIR)
-	robot merge --input $(DEFAULT_BUILD_DIR)/snomed-parts.owl --output $@
+# Intermediate file: We don't actually care about full SNOMED enhanced by LOINC-SNOMED Ontology. This is just a temporary step to be used in `robot extract`
+# todo: warning: error could occur in the future, because I don't know what caused this to resolve in the first place. very strange; this error occurred the very first time I ran it. It ran as a prereq to make output/tmp/subclass-rels-loinc-snomed.tsv. Could not replicate. Next try succeeded without changing any input files. The only thing I did was added -vvv to the end, but this should only print debugging info.
+#make output/tmp/subclass-rels-loinc-snomed.tsv
+#
+#robot merge --input output/analysis/snomed/snomed-unreasoned.ofn --input output/analysis/loinc-snomed/loinc-snomed-module.ofn --output output/analysis/loinc-snomed/snomed-with-loinc-snomed-full-unreasoned.owl
+#INVALID ELEMENT ERROR "http://snomed.info/id/1295447006
+#http://snomed.info/id/1295449009
+#http://snomed.info/id/1295448001" contains invalid characters
+#For details see: http://robot.obolibrary.org/errors#invalid-element-error
+#Use the -vvv option to show the stack trace.
+#Use the --help option to see usage information.
+#make: *** [output/analysis/loinc-snomed/snomed-with-loinc-snomed-full-unreasoned.owl] Error 1
+#
+#
+#make output/tmp/subclass-rels-loinc-snomed.tsv
+#
+#robot query ...
+$(LOINC_SNOMED_OWL_DIR)/snomed-with-loinc-snomed-full-unreasoned.owl: $(SNOMED_OWL_DIR)/snomed-unreasoned.ofn $(LOINC_SNOMED_OWL_DIR)/loinc-snomed-module.ofn
```

Comment:
FYI, I got an `INVALID ELEMENT ERROR` the first time I ran this, but it immediately worked the second time.